src/components/inline/Card/Card.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from "prop-types";
+import React from "react";
+import s from "./Card.module.css";
+
+const Card = ({ photo, title, description }) => (
+  <div className={s["card-container"]}>
+    <div className={s["img-container"]}>
+      <img className={s.img} src={photo} alt="" />
+    </div>
+    <div>
+      <div className={s["title-container"]}>
```

Comment:
Regarding these `title-container` and `description-container` elements: although I think it's a good thing in general to separate the concerns of styling from positioning, I think this is a case where it's not necessary because the components are simple enough.

Also note that in this case, because you are using `<div>`s for both the container and the inner element, they are both block elements and the nesting of block elements doesn't offer us much benefit. A container element would still be beneficial in the case where you have non-block elements, such as a button (which would normally be inline) or an img (which is basically an inline-block element).